docs/website/docs/command-reference/create.md

```diff
@@ -53,8 +53,6 @@ odo create nodejs --starter nodejs-starter
 
 This will download the example template corresponding to the chosen component type (in the example above, `nodejs`) in your current directory (or the path provided with the `--context` flag).
 
-If a starter project has its own devfile, then this devfile will be preserved.
```

Comment:
All these modified files in `docs/website/docs/` comes unmodified from the v2 branch, right?